app/src/main/res/layout/item_layout.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="8dp"
+    app:cardCornerRadius="32dp"
+    app:cardElevation="4dp">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="#E9EBEC"
+        android:padding="16dp">
+
+        <ImageView
+            android:id="@+id/item_icon"
+            android:layout_width="64dp"
+            android:layout_height="64dp"
+            android:layout_marginTop="16dp"
+            android:src="@drawable/ic_sunny"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/weatherC"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="26°"
```

Comment:
Preview'de gozukmesini istedigin datalari tools:text="26" kullanmakta fayda var. Aksi durumda datanin setlenmedigi durumlarda tum liste itemlarinda 26 gozukecektir.

Bu yorum bu xml'deki android:text ve android:src icin de gecerli

Reply:
Hocam tools:text kullandığım textler ve imgler emulatörde gözükmüyor bunun için yapmam gereken ekstra bir şey var mı